spring-batch-samples/README.md

```diff
@@ -29,7 +29,7 @@ Job/Feature                                       | skip | retry | restart | aut
 [compositeItemWriterSample](#compositeItemWriter) |      |       |         |                   |               |            |     X      |              |                |                |           
 [customerFilter](#customerFilter)                 |      |       |         |                   |               |            |            |              |                |                |     X     
 [delegating](#delegating)                         |      |       |         |                   |               |            |     X      |              |                |                |           
-[football](#football)                             |      |       |         |                   |               |            |            |              |                |                |           
+[football](#football-job)                             |      |       |         |                   |               |            |            |              |                |                |           
```

Comment:
Yeah, no problem
It's done in the following commit :
https://github.com/spring-projects/spring-batch/pull/413/commits/28c6ab0c21faa6a224d67ca892a69be9a25431b6